core/src/test/java/com/alibaba/fescar/core/protocol/transaction/GlobalBeginResponseTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *  Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.alibaba.fescar.core.protocol.transaction;
+
+import com.alibaba.fescar.core.protocol.ResultCode;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * A unit test for {@link GlobalBeginResponse}
+ *
+ * @author liujc
+ * @date 2019/3/22 11:09
+ **/
+public class GlobalBeginResponseTest {
+    private final String XID = "test_xid";
+    private final String EXTRA_DATA = "test_extra_data";
+    private final ResultCode RESULT_CODE = ResultCode.Success;
+
+    @Test
+    public void testGetSetXid() {
+        GlobalBeginResponse globalBeginResponse = new GlobalBeginResponse();
+        globalBeginResponse.setXid(XID);
+        Assert.assertEquals(XID, globalBeginResponse.getXid());
+    }
+
+    @Test
+    public void testGetSetExtraData() {
+        GlobalBeginResponse globalBeginResponse = new GlobalBeginResponse();
+        globalBeginResponse.setExtraData(EXTRA_DATA);
+        Assert.assertEquals(EXTRA_DATA, globalBeginResponse.getExtraData());
+    }
+
+    @Test
+    public void testGetTypeCode() {
+        GlobalBeginResponse globalBeginResponse = new GlobalBeginResponse();
+        Assert.assertEquals(GlobalBeginResponse.TYPE_GLOBAL_BEGIN_RESULT, globalBeginResponse.getTypeCode());
+    }
+
+    @Test
+    public void testDoEncodeAndDeode() {
+        GlobalBeginResponse globalBeginResponse = new GlobalBeginResponse();
+        globalBeginResponse.setXid(XID);
+        globalBeginResponse.setExtraData(EXTRA_DATA);
+        globalBeginResponse.setResultCode(RESULT_CODE);
+        globalBeginResponse.doEncode();
```

Comment:
should encoded into byte[], decoded by another instance, comparing the attributes are equal？

Reply:
Ok, I'll revise it here.